auth-backends/src/main/scala/bootstrap/rudder/plugin/AuthBackendsConf.scala

```diff
@@ -285,6 +288,8 @@ class AuthBackendsSpringConfiguration extends ApplicationContextAware {
     }
   }
 
+  @Bean def roleApiMapping = RudderConfig.roleApiMapping
+
```

Comment:
This need to be changed to 'new RoleApiMapping' b/c if not, it needs a new version of rudder-core.
we will do that full correction in the backport (and then merge it)